usr/sbin/revpi-bthelper

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth helper for Revolution Pi BT
+. /usr/bin/revpi-common
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+	echo "Usage: $0 <bluetooth hci device>"
+	exit 0
+fi
+
+dev=$1
+
+# Need to bring hci up before looking at MAC as it can be all zeros during init
+# TODO: address prefix for Revpi devices needed.
+if ! /bin/hciconfig "$dev" | grep -q "BD Address: B8:27:EB"; then
+	echo Not UART-attached BT Modem on Revolution Pi
```

Comment:
What is it with the "on Revolution Pi"? Just drop this part.

Reply:
ok

Reply:
Again, drop " on Revolution Pi". It doesn't make any sense in this context.